### ISSUE TITLE
Keep open for multiple selection

### ISSUE THREAD

**Issue by thdxr** [thisbeyond/solid-select#27]
Right now the component closes as soon as you pick an option. But for situations where multiple values are expected to be selected it would be nice to keep it open until explicitly clicked outside

**Comment by martinpengellyphillips** [comment]
Yeah, I think I need to separate some of the assumed core logic to provide more flexibility here.
